fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -72,7 +78,7 @@ fun TabItem(
         tabItemTokens
             ?: FluentTheme.controlTokens.tokens[ControlTokens.ControlType.TabItemControlType] as TabItemTokens
 
-    val tabItemInfo = TabItemInfo(textAlignment, style)
```

Comment:
Also we need color for icon as well

Reply:
This PR is for TabItem Indicator only, for Icon we will have to modify Icon class as well to support a Brush object, will add that in the next PR.